suffuse/util.go

```diff
@@ -97,3 +97,9 @@ func ScratchFile() Path {
   deleteOnExit = append(deleteOnExit, path)
   return path
 }
+
+// https://github.com/golang/go/issues/1312
+func FileExists(path string) bool {
+  _, err := os.Stat(path)
```

Comment:
Since it's called FileExists this is probably good as-is, but something to always think about is whether it should be a call to Stat or a call to Lstat. I would prefer to see methods like this on Path for a bunch of reasons (avoiding having to import "os" everywhere for instance, encapsulating usage of "os" package, type safety...) The reason Path doesn't have an Exists() method is because I couldn't believe calling Stat and checking for error was the "right" way. But it seems that it is as right as we have available. In any case can you move this into Path?